README.md

```diff
@@ -77,6 +77,10 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.3/install.html#
 | `bbb_monitoring_all_in_one_htpasswd_user` | The user for the htpasswd - _(required)_ if external | `Undefined` | |
 | `bbb_monitoring_all_in_one_htpasswd` | The password for the htpasswd - _(required)_ if external | `Undefined` | |
 | `bbb_monitoring_recordings_from_disk` | Collect recordings metrics by querying the disk instead of the API. See [this](https://bigbluebutton-exporter.greenstatic.dev/exporter-user-guide/#optimizations) for details. | `true` |
+| `bbb_monitoring_systemd_enable` | Deploy monitoring as systemd service(not recommended) | `false` | Works only when `bbb_monitoring_all_in_one_enable` is `false` |
```

Comment:
Maybe adding a check to:
https://github.com/ebbba-org/ansible-role-bigbluebutton/blob/bac8ed7f0c1c8f12df00b45d63d9c3cd5fe5d39b/tasks/checkvars.yml#L3
would be nice so the role fails fast when both vars are set.

Reply:
> Would like to see this tested.

I think I'll have some time later on today or tomorrow to give it a test.

Reply:
Thanks @BrutalBirdie, I added the check to checkvars.yml